provider/common.go

```diff
@@ -37,10 +37,11 @@ type PostOrderOpts struct {
 }
 
 type RPCOpts struct {
-	Endpoint   string
-	Timeout    time.Duration
-	PrivateKey *solana.PrivateKey
-	AuthHeader string
+	Endpoint       string
+	Timeout        time.Duration
+	PrivateKey     *solana.PrivateKey
+	AuthHeader     string
+	CacheBlockHash time.Duration
```

Comment:
should also have a parameter to enable this. by default, the `recentBlockHashStore` should not run

Reply:
agree, changed